src/lib/containers/AccessApprovalsTable.tsx

```diff
@@ -59,69 +59,65 @@ export const AccessApprovalsTable: React.FunctionComponent<
         Status in Access Requirements
       </Typography>
       <Table striped borderless bordered={false}>
-        <thead className="access-approval-header">
```

Comment:
Just dropped unnecessary class names and reorganized structure to match AccessRequirementTable (and match the now-shared CSS)